CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.4.0 (April 6, 2021)
```

Comment:
I think we want `(Unreleased)` here for the date, and then the TeamCity job takes care of replacing with the date